aws-ts-static-website/index.ts

```diff
@@ -0,0 +1,210 @@
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+import { Output } from "@pulumi/pulumi";
+
+import * as fs from "fs";
+import * as mime from "mime";
+import * as path from "path";
+
+// Load the Pulumi program configuration. These act as the "parameters" to the Pulumi program,
+// so that different Pulumi Stacks can be brought up using the same code.
+
+const stackConfig = new pulumi.Config("static-website");
+const config = {
+    // pathToWebsiteContents is a relativepath to the website's contents.
+    pathToWebsiteContents: stackConfig.require("pathToWebsiteContents"),
+    // targetDomain is the domain/host to serve content at.
+    targetDomain: stackConfig.require("targetDomain"),
+    // ACM certificate for the target domain. Must be in the us-east-1 region.
+    certificateArn: stackConfig.require("certificateArn"),
+};
+
+// contentBucket is the S3 bucket that the website's contents will be stored in.
+const contentBucket = new aws.s3.Bucket(
+    "contentBucket",
+    {
+        bucket: config.targetDomain,
+        acl: "public-read",
+        // Configure S3 serve bucket contents as a website. This way S3 will automatically convert
+        // requests for "foo/" to "foo/index.html".
+        website: {
+            indexDocument: "index.html",
+            errorDocument: "404.html",
+        }
+    });
```

Comment:
Not a big deal, but the formatting of these calls is different than we use in all other examples in this repo (and elsewhere).

```typescript
const contentBucket = new aws.s3.Bucket("contentBucket", {
    bucket: config.targetDomain,
    acl: "public-read",
    // Configure S3 serve bucket contents as a website. This way S3 will automatically convert
    // requests for "foo/" to "foo/index.html".
    website: {
        indexDocument: "index.html",
        errorDocument: "404.html",
     }
});
```

Reply:
Done. I missed this in the PR, so I'll just make the commit directly into `master`, like some kind of animal.